src/posts/memcached-fms.md

```diff
@@ -38,7 +38,7 @@ for item in content_items:
 ```
 This way, while we'd have to wait for a single memcache fetch that we didn't before, at least it would only be a single one.  In some cases, refactoring the code would have been difficult, but we could at least pre-fetch the items we'd need, and keep them in memory until the end of the request, similarly avoiding a sequential fetch.
 
-This was a pretty effective method &ndash; on our first round of testing, half of the hundred URLs we hit were at least 50% slower at the 50th percentile, and `khanacademy.org/math` was over 7 times slower!  But after several rounds of fixes and new tests, we were able to get the new version faster on average and on most specific URLs.  Unfortunately, this didn't cover everything &ndash; some real user traffic is harder to simulate, and because caching is less effective on test versions with little traffic, there was a lot of noise in our small sample, and so small changes in performance were hard to spot, even if they affected very very popular routes where that small difference would add up.
+This was a pretty effective method &ndash; on our first round of testing, half of the hundred URLs we hit were at least 50% slower at the 50th percentile, and `khanacademy.org/math` was over 7 times slower!  But after several rounds of fixes and new tests, we were able to get the new version faster on average and on most specific URLs.  Unfortunately, this didn't cover everything &ndash; some real user traffic is harder to simulate, and because caching is less effective on test versions with little traffic, there was a lot of noise in our small sample, and so small changes in performance were hard to spot, even if they affected very popular routes where that small difference would add up.
```

Comment:
What's the difference here?  -- github isn't highlighting anything.

Reply:
Ah, I see: s/very very/very/.  As below, the doubling is intentional here and this should be reverted.